cheddar/resources/service.py

```diff
@@ -9,6 +9,7 @@ class Service(object):
     EPLATBY = "eplatby"
     TATRAPAY = "tatrapay"
     CARDPAY = "cardpay"
+    COMFORTPAY = "comfortpay"
```

Comment:
This should also be mentioned in [README](https://github.com/backbonesk/cheddar-python/blob/master/README.md) in "Creating a transaction" section.